### PR TITLE
added to Nav

### PR DESCRIPTION
**Pull request by Grgkalunta**

### CODE REVIEW

File: LiveBlip/Components/Layout/NavMenu.razor [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <div class="container-fluid px-3">
 	<nav class="navbar navbar-expand justify-content-between">
-		<NavLink class="navbar-brand" href="#">LiveBlip</NavLink>
+		<NavLink class="navbar-brand" href="#"><img width= "150" height= "70" src="Images\LiveBlipLogo.png"/></NavLink>
 
 		<div class="navbar-nav justify-content-end">
-			<NavLink class="nav-link" href="#">Home</NavLink>
```
Comment (CodePidgy) [inline LiveBlip/Components/Layout/NavMenu.razor:6]: Why did you change this link? It was perfect the way it was

**Comment by Hybrid-Codes** [comment]
Why did you remove home from the nav bar?

File: LiveBlip/Components/Layout/NavMenu.razor [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <div class="container-fluid px-3">
 	<nav class="navbar navbar-expand justify-content-between">
-		<NavLink class="navbar-brand" href="#">LiveBlip</NavLink>
+		<NavLink class="navbar-brand" href="#"><img width= "150" height= "70" src="Images\LiveBlipLogo.png"/></NavLink>
```
Comment (CodePidgy) [inline LiveBlip/Components/Layout/NavMenu.razor:3]: Make the width 200, 150 is too squashed

**Review by CodePidgy — CHANGES_REQUESTED** [review]

**Comment by Grgkalunta** [comment]
when you select start it will take you to home page

File: LiveBlip/Components/Layout/NavMenu.razor (hunk 2 above)
Comment (Grgkalunta) [reply LiveBlip/Components/Layout/NavMenu.razor:3]: sure

**Comment by CodePidgy** [comment]
You're not listening. We need a search bar of sorts that will take in the device IMEI number. If you don't know what that is, Google it. Once entered, it will take the user to the Home page. You also have not fixed those other things

**Comment by Grgkalunta** [comment]
wait

**Comment by Grgkalunta** [comment]
Why does a welcome page need to do that?

**Comment by CodePidgy** [comment]
Because that's how this is going to work. This is how we planned the website to work since the very beginning

**Comment by Grgkalunta** [comment]
I was doing the welcome page not home page bro

**Comment by CodePidgy** [comment]
Greg! The Welcome page greets the user with the search bar, since if they don't have an account they need to be able to track their device without an account simply by entering the device IMEI

**Comment by Grgkalunta** [comment]
Stop typing G

**Comment by CodePidgy** [comment]
Excuse me?

**Comment by CodePidgy** [comment]
Even the prototype has the search bar in it. You were there when we were presenting it to ma'am. You were there when we explained the purpose of it
